### ISSUE TITLE
redis-rdb-tools access on web browser 

### ISSUE THREAD

**Issue by vinod-sisodiya** [sripathikrishnan/redis-rdb-tools#148]
Hi,

how can i see web-base control panel for Redis-server , Please help me out.

**Comment by oranagra** [comment]
@vinod-sisodiya 
if you're talking about https://rdbtools.com/ then i think you're might be asking in the wrong place.

**Comment by vinod-sisodiya** [comment]
Yes .!! i m talking about https://rdbtools.com/. is it cluster monitoring  tools or not?
